### PR TITLE
[CMSP-992] plugin documentation update

### PR DESCRIPTION
**Pull request by jazzsequence**
This PR updates the documentation for the PAPC 2.0 changes.

Forked from #272

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,20 @@ add_filter( 'pantheon_purge_post_type_ignored', 'filter_ignored_posts' );
 
 This will prevent the cache from being purged if the given post type is updated.
 
+### Setting the Cache Max Age with a filter
+
+The cache max age setting is controlled by the [Pantheon Page Cache]() admin page. As of 2.0.0, there are three cache age options by default -- 1 week, 1 month, 1 year. Pantheon Advanced Page Cache automatically purges the cache of updated and related posts and pages, but you might want to override the cache max age value and set it programmatically. In this case, you can use the `pantheon_cache_default_max_age` filter added in [Pantheon MU plugin 1.4.0+](https://docs.pantheon.io/guides/wordpress-configurations/wordpress-cache-plugin#override-the-default-max-age). For example:
```
Comment (pwtyler) [inline README.md]: Looks like you accidentally a link here?

Comment (pwtyler) [inline README.md]:
```suggestion
The cache max age setting is controlled by the [Pantheon Page Cache]() admin page. As of 2.0.0, there are three cache age options by default — 1 week, 1 month, 1 year. Pantheon Advanced Page Cache automatically purges the cache of updated and related posts and pages, but you might want to override the cache max age value and set it programmatically. In this case, you can use the `pantheon_cache_default_max_age` filter added in [Pantheon MU plugin 1.4.0+](https://docs.pantheon.io/guides/wordpress-configurations/wordpress-cache-plugin#override-the-default-max-age). For example:
```
fixed width suggestion obfuscates it a bit but I promise that's an em dash 😁

File: README.md [hunk 2]
```diff
@@ -356,7 +392,9 @@ See [CONTRIBUTING.md](https://github.com/pantheon-systems/pantheon-advanced-page
...
+* Adds new admin alerts and Site Health tests about default cache max age settings and recommendations [[#268](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/268), [#271](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/271)]. The default Pantheon GCDN cache max age value has been updated to 1 week in the [Pantheon MU plugin](https://github.com/pantheon-systems/pantheon-mu-plugin). For more information, see the [release note](https://docs.pantheon.io/release-notes/2024/04/pantheon-mu-plugin-1-4-0-update).
+* Updated UI in Pantheon Page Cache admin page when used in a Pantheon environment (with the Pantheon MU plugin). [[#272](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/272)]
+* Automatically updates the cache max age to the recommended value (1 week) if it was saved at the old default value (600 seconds). [[#269](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/269)]
```
Comment (pwtyler) [inline README.md:397]: > Automatically updates the cache max age to the recommended value

This should probably be an upgrade notice as well.

File: readme.txt [hunk 3]
```diff
@@ -150,6 +150,16 @@ By default, Pantheon Advanced Page Cache is pretty aggressive in how it clears i
 
 This will prevent the cache from being purged if the given post type is updated.
 
+= Setting the Cache Max Age with a filter =
+
+The cache max age setting is controlled by the [Pantheon Page Cache]() admin page. As of 2.0.0, there are three cache age options by default -- 1 week, 1 month, 1 year. Pantheon Advanced Page Cache automatically purges the cache of updated and related posts and pages, but you might want to override the cache max age value and set it programmatically. In this case, you can use the `pantheon_cache_default_max_age` filter added in [Pantheon MU plugin 1.4.0+](https://docs.pantheon.io/guides/wordpress-configurations/wordpress-cache-plugin#override-the-default-max-age). For example:
```
Comment (pwtyler) [inline readme.txt]:
```suggestion
The cache max age setting is controlled by the [Pantheon Page Cache]() admin page. As of 2.0.0, there are three cache age options by default— 1 week, 1 month, 1 year. Pantheon Advanced Page Cache automatically purges the cache of updated and related posts and pages, but you might want to override the cache max age value and set it programmatically. In this case, you can use the `pantheon_cache_default_max_age` filter added in [Pantheon MU plugin 1.4.0+](https://docs.pantheon.io/guides/wordpress-configurations/wordpress-cache-plugin#override-the-default-max-age). For example:

```
Same as above, also need link.

File: readme.txt [hunk 4]
```diff
@@ -321,6 +331,24 @@ Setting surrogate keys for posts with large numbers of taxonomies (such as WooCo
 	}
 	add_filter('pantheon_should_add_terms', 'custom_should_add_terms', 10, 2);
 
+== Other Filters ==
+
+= `pantheon_apc_disable_admin_notices` =
```
Comment (pwtyler) [inline readme.txt:336]: Does backticks in heading work on wp.org readmes?

File: README.md (hunk 2 above)
Comment (jazzsequence) [reply README.md:397]: Oh good call. Yes it should.

File: readme.txt (hunk 4 above)
Comment (jazzsequence) [reply readme.txt:336]: Yes https://wordpress.org/plugins/pantheon-advanced-page-cache/

Comment (jazzsequence) [reply readme.txt:336]: Wait, in heading maybe not. I guess we'll find out. I couldn't find anything definitive.

**Review by pwtyler — APPROVED** [review]

**Pull request merged**